app/forms/idv/api_image_upload_form.rb

```diff
@@ -472,5 +470,9 @@ def store_failed_images(client_response, doc_pii_response)
     def image_resubmission_check?
       IdentityConfig.store.doc_auth_check_failed_image_resubmission_enabled
     end
+
+    def liveness_checking_enabled?
+      params[:liveness_checking_enabled] == 'true'
+    end
```

Comment:
I don't think we'd want to trust validation based on a user-provided parameter?

Reply:
@aduth and I caught up on this and there is a guard in TrueIDRequest